### PR TITLE
Add support for the creation of Receipts

### PR DESCRIPTION
**Pull request by onevision**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,3 +36,16 @@ Currently, only the private app authentication method has been implemented. We u
 The SBT build tool must be installed and [Xero's XML schema library](https://github.com/XeroAPI/XeroAPI-Schemas) must be checked out as a sibling project. Build with:
 
     sbt compile
+
+## Examples
+
+### Creating an invoice
```
Comment (benmccann) [inline README.md]: What if we put this in the "Example Usage" section instead?

File: client/src/main/java/com/connectifier/xeroclient/XeroClient.java [hunk 2]
```diff
@@ -128,6 +128,7 @@ protected ResponseType get(String endPoint, Date modifiedAfter, Map<String,Strin
   protected ResponseType put(String endPoint, JAXBElement<?> object) {
     OAuthRequest request = new OAuthRequest(Verb.PUT, BASE_URL + endPoint);
     String contents = marshallRequest(object);
+    System.out.println(contents);
```
Comment (benmccann) [inline client/src/main/java/com/connectifier/xeroclient/XeroClient.java]: We probably shouldn't check this in

File: client/src/main/java/com/connectifier/xeroclient/XeroClient.java [hunk 3]
```diff
@@ -309,6 +310,10 @@ public Invoice getInvoice(String id) {
   public List<Invoice> createInvoice(Invoice invoice) {
     return put("Invoices", objFactory.createInvoice(invoice)).getInvoices();
   }
+  
+  public List<Receipt> createReceipt(Receipt receipt) {
+	    return put("Receipts", objFactory.createReceipt(receipt)).getReceipts();
```
Comment (benmccann) [inline client/src/main/java/com/connectifier/xeroclient/XeroClient.java]: replace the tabs with spaces

**Comment by onevision** [comment]
Hi Ben

Are you happy to accept this pull request now I have made the changes you requested?

thanks

Richard

**Pull request merged**